src/it/resources/com/google/checkstyle/test/chapter5naming/rule523methodnames/InputMethodName.java

```diff
@@ -42,6 +42,9 @@ void testingFoo_foo() {}
   @Test
   void testing_0123() {}
 
+  @Test
+  void Testing_Foo_False_Negative() {} // false negative, should be a violation
```

Comment:
This is a false negative. If there's `_` in a test method name then we're completely ignoring the method. Is there a workaround for this? @romani

Reply:
See below.
https://github.com/checkstyle/checkstyle/pull/15515#discussion_r1720518410